lib/OpenQA/WebAPI.pm

```diff
@@ -156,10 +157,11 @@ has secrets => sub {
 sub startup {
     my $self = shift;
 
+    $self->_read_config;
+
     # Set some application defaults
-    $self->defaults(appname => 'openQA');
+    $self->defaults(appname => $self->app->config->{'global'}->{'appname'});
```

Comment:
I'll play @coolo here :).
These quotes are not needed there and we are gradually dropping them. Like on the line below.